src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker.cs

```diff
@@ -295,12 +333,11 @@ internal void RemoveProject(AbstractProject project)
 
             UpdateProjectBinPath(project, project.BinOutputPath, null);
 
-            using (Dispatcher.CurrentDispatcher.DisableProcessing())
```

Comment:
not sure what make disable processing to removed and what make it to left out.

Reply:
I'm somewhat convinced it doesn't do anything. This was added to prevent some reentrancy when we take locks...